src/gui/presets.cpp

```diff
@@ -2251,6 +2297,20 @@ void FurnaceGUI::initSystemPresets() {
       ) // software controlled 8 bit DAC
     }
   );
+  ENTRY(
+    "Alpha denshi Equites", {
+      CH(DIV_SYSTEM_MSM5232, 64, 0, ""), // // clock speed should be 6.144 MHz
```

Comment:
You can set custom clock rates with the customClock flag (in Hz). For example: `"customClock=6144000"`